lib/beaker_puppet_helpers/dsl.rb

```diff
@@ -224,5 +224,30 @@ def fact_on(host, name, opts = {})
     def fact(name, opts = {})
       fact_on(default, name, opts)
     end
+
+    # Show if bolt package is available
+    #
+    # @param [Beaker::Host] host
+    # @return Boolean True if package is available.
```

Comment:
This needs to be 
```suggestion
    # @return [Boolean] True if package is available.
```

Or, if you follow the suggestion below it's implicit
```suggestion
    # @return True if package is available.
```

And yes, I also just noticed that for `fact_on` it's also wrong.